### ISSUE TITLE
Add installer

### ISSUE THREAD

**Issue by BalintFarkas** [mcmonkeyprojects/KeyboardChatterBlocker#9]
Hey!

First of all, thanks for picking up the slack! I bought a mech keyboard at the beginning of this year and I was stuck with the two unmaintained solutions you also mentioned. Today, out of frustration, I did a fresh GitHub search, and was very happy to see a new, maintained project!

Second, I think you'd benefit a lot from having a MSI or similar Windows installer. Your user story is already pretty cool; however, downloading an EXE and manually copying it somewhere is a bit of a kludge.

Third, I used to develop a lot of Windows frontend stuff, so I can do this for you if you don't have the time. I'd simply use Visual Studio's own Setup Project as described here: https://medium.com/@kanchanaj/how-to-create-a-windows-installer-setup-file-for-desktop-application-using-visual-studio-2017-beb13015fb58

Cheers!
balint

**Comment by mcmonkey4eva** [comment]
I've always personally held the opinion that installers were the awkward kludge, at least for small applications where an installer doesn't do anything more than copy an .exe to a folder. Installers also tend to add weird limits (admin privilege requirements, can't have configs and executables side-by-side, ...) that don't affect normal programs.

That said, that's just my opinion and this issue being posted means clearly at least one person disagrees, that's probably good enough reason to add an installer as an option for those who prefer them.

That article explaining VS's own option for it seems like a pretty decent way to go about it, thank you for that link.

I've added a `.msi` installer. As part of being able to support that, I also added logic to redirect the config file into `%localappdata%` when the program has been installed to program files (but still otherwise can live next to the executable when not in that restricted path).

This is available in release 1.06 @ https://github.com/mcmonkeyprojects/KeyboardChatterBlocker/releases
